README.md

```diff
@@ -1,6 +1,17 @@
 # Open-Telemetry Operations Exporters for Go
 
-Provides OpenTelemetry Exporters for Google Cloud Operations. 
+This repository contains the source code of 2 packages of OpenTelemetry exporters to [Google Cloud Trace](https://cloud.google.com/trace) and [Google Cloud Monitoring](https://cloud.google.com/monitoring).
 
-## Installation
+## OpenTelemetry Google Cloud Trace Exporter
 
+OpenTelemetry Google Cloud Trace Exporter allow the user to send collected traces and spans to Google Cloud.
+
+[Google Cloud Trace](https://cloud.google.com/trace) is a distributed tracing backend system. It helps developers to gather timing data needed to troubleshoot latency problemts in microservice architecture as well as monolithic architecture. It manages both the collection and lookup of gathered trace data.
```

Comment:
```suggestion
[Google Cloud Trace](https://cloud.google.com/trace) is a distributed tracing backend system. It helps developers to gather timing data needed to troubleshoot latency problems in microservice & monolithic architectures. It manages both the collection and lookup of gathered trace data.
```

Reply:
Probably should replace this paragraph with a link to the trace/README.md file, as is done for metrics below? (currently this paragraph is replicated in both files)

Reply:
I agree. I'll move the 3rd paragraph to Trace Exporter README